main.py

```diff
@@ -133,6 +125,18 @@ def get_jenkins_jobs(server, job_search_fields):
 
 	return relevant_jobs
 
+def get_bugs_set(blockers_file):
```

Comment:
```suggestion
def get_bugs_set(blockers):
```
Small change since technically the blocker file object isn't being passed; the loaded YAML is